azkaban-exec-server/src/test/java/azkaban/execapp/JobRunnerTest.java

```diff
@@ -317,8 +316,7 @@ public void testDelayedExecutionCancelledJob() throws InterruptedException {
     Assert.assertTrue(logFile.exists());
 
     // wait so that there's time to make the "DB update" for KILLED status
-    Awaitility.await().atMost(10L, TimeUnit.SECONDS).pollInterval(10L, TimeUnit.MILLISECONDS)
-        .until(() -> loader.getNodeUpdateCount("testJob"), is(2));
+    TestUtils.await().until(() -> loader.getNodeUpdateCount("testJob"), is(2));
```

Comment:
It would be good to use assertJ here as well. It can be done in another PR.

Reply:
Do you know out of your head how that would be done? If not never mind I could look it up.. Thanks for the great review once again 👍

Reply:
Tracked in the issue #1471